lib/DDGC/Schema/Result/ActivityFeed.pm

```diff
@@ -0,0 +1,68 @@
+package DDGC::Schema::Result::ActivityFeed;
+# ABSTRACT: Activity feed reault class
+
+use Moo;
+extends 'DDGC::Schema::Result';
+use DBIx::Class::Candy;
+use DDGC::Util::Markup;
+
+table 'activity_feed';
+
+column id => {
+    data_type => 'bigint',
+    is_auto_increment => 1,
+};
+primary_key 'id';
+
+column created => {
+    data_type => 'timestamp with time zone',
+    set_on_create => 1,
+};
+
+column description => {
+    data_type => 'text',
+    is_nullable => 0,
+};
+
+column format => {
+    data_type => 'text',
+    default_value => 'markdown',
+};
+
+column for_user => {
+    data_type => 'bigint',
+    is_nullable => 1,
+};
+
+column for_role => {
+    data_type => 'int',
+    is_nullable => 1,
+};
+
+has renderer => (
+    is => 'ro',
+    lazy => 1,
+    builder => '_build_renderer',
+);
+sub _build_renderer {
+    DDGC::Util::Markup->new;
+}
+
+sub render_description{
```

Comment:
Thanks.

[DDGC::Util::Markup](https://github.com/duckduckgo/community-platform/blob/0dc490e/lib/DDGC/Util/Markup.pm) is intended to replace [DDGC::Markup](https://github.com/duckduckgo/community-platform/blob/0dc490e/lib/DDGC/Markup.pm) - I think it's about 90% ready - little bit of clean up and some rendering quirks to fix.